package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "author": "Andrew Balmos <andrew@balmos.org> (https://github.com/abalmos)",
   "license": "Apache 2.0",
+  "scripts": {
+    "test": "./istantestd.sh"
```

Comment:
Should there be a `cover` and `test` option?

One uses istanbul and generates the coverage reports and the other just runs the tests?  @awlayton what say you?